src/Azure.Functions.ArtifactAssembler/ArtifactAssembler.cs

```diff
@@ -181,11 +208,117 @@ private async Task CreateVisualStudioCoreToolsAsync()
             });
 
             await Task.WhenAll(packTasks);
+            Console.WriteLine($"Finished assembling Visual Studio Core Tools artifacts");
+        }
+
+        private async Task CreateCliCoreToolsAsync()
+        {
+            Console.WriteLine($"Starting to assemble CLI Core Tools artifacts");
```

Comment:
Can you remove the $ from all strings that are not doing string interpolation?

Reply:
done!